vendor/github.hpe.com/hpe/hpc-shastarelm-release/lib/install.sh

```diff
@@ -244,6 +244,42 @@ function skopeo-sync() {
         /image "$NEXUS_REGISTRY"
 }
 
```

Comment:
@ekamin-hpe since it looks like you made this change to the upstream hpc-shastarelm-release repository, this change should be pulled in using 'git vender update' with CSM instead of manually editing the vendored file. If you can remove this update from your CSM branch, I will do the "git vendor update release main" with the CSM branch later before I do the build.

Reply:
Talked about this in Slack, but just for posterity I did actually use `git vendor update` after making the change to hpc-shastarelm-release (the result is 9de2c97f92f43c4c4683315949dfb41ccaa6a7f2, the second commit).